--- IO.cpp	2020-11-06 05:20:56.551613790 -0500
+++ IO.cpp	2020-11-07 05:21:08.978132118 -0500
@@ -171,10 +171,10 @@
 #if !defined(USE_ALTERNATE_FM_LEDS)
   setFMInt(false);
 #endif
-  setDStarInt(true);
+  setDMRInt(true);
 
   delayInt(250);
-  setDMRInt(true);
+  setDStarInt(true);
 
   delayInt(250);
   setYSFInt(true);
@@ -217,10 +217,10 @@
   setYSFInt(false);
 
   delayInt(250);
-  setDMRInt(false);
+  setDStarInt(false);
 
   delayInt(250);
-  setDStarInt(false);
+  setDMRInt(false);
 #endif
 }
 
